js/app.js

```diff
@@ -70,6 +100,11 @@
    *   and passing two aditional arguments, event and 'light-green'
    */
 
+  let btn4Args = [event, "light-green"];
```

Comment:
btn4 passes because `event` is not used. You would not want to reference `event` outside of the callback. You will either get an error or reference something you are not expecting.